Source/Core/Core/HW/DVD/AMMediaboard.h

```diff
@@ -0,0 +1,221 @@
+  DIMMMemory2 = 0xFF000000,
+
+  DIMMExtraSettings = 0x1FFEFFE0,
+
+  NetworkControl = 0xFFFF0000,
+
+  DIMMCommandVersion1 = 0x1F900000,
+  DIMMCommandVersion2 = 0x84000000,
+  DIMMCommandVersion2_2 = 0x89000000,
+  DIMMCommandExecute2 = 0x88000000,
+
+  NetworkCommandAddress = 0x1F800200,
+  NetworkCommandAddress2 = 0x89040200,
+
+  NetworkBufferAddress1 = 0x1FA00000,
+  NetworkBufferAddress2 = 0x1FD00000,
+  NetworkBufferAddress3 = 0x89100000,
+  NetworkBufferAddress4 = 0x89180000,
+  NetworkBufferAddress5 = 0x1FB00000,
+
+  FirmwareAddress = 0x84800000,
+
+  FirmwareMagicWrite1 = 0x00600000,
+  FirmwareMagicWrite2 = 0x00700000,
+};
+
+// Mario Kart GP2 has a complete list of them
+// but in japanese
+// They somewhat match WSA errors codes
+enum SocketStatusCodes
+{
+  SSC_E_4 = -4,  // Failure (abnormal argument)
+  SSC_E_3 = -3,  // Success (unsupported command)
+  SSC_E_2 =
+      -2,  // Failure (failed to send, abnormal argument, or communication condition violation)
+  SSC_E_1 = -1,  // Failure (error termination)
+
+  SSC_EINTR = 4,    // An interrupt occurred before data reception was completed
+  SSC_EBADF = 9,    // Invalid descriptor
+  SSC_E_11 = 11,    // Send operation was blocked on a non-blocking mode socket
+  SSC_EACCES = 13,  // The socket does not support broadcast addresses, but the destination address
+                    // is a broadcast address
+  SSC_EFAULT =
+      14,  // The name argument specifies a location other than an address used by the process.
+  SSC_E_23 = 23,     // System file table is full.
+  SSC_AEMFILE = 24,  // Process descriptor table is full.
+  SSC_EMSGSIZE =
+      36,  // Socket tried to send message without splitting, but message size is too large.
+  SSC_EAFNOSUPPORT = 47,   // Address prohibited for use on this socket.
+  SSC_EADDRINUSE = 48,     // Address already in use.
+  SSC_EADDRNOTAVAIL = 49,  // Prohibited address.
+  SSC_E_50 = 50,           // Non-socket descriptor.
+  SSC_ENETUNREACH = 51,    // Cannot access specified network.
+  SSC_ENOBUFS = 55,        // Insufficient buffer
+  SSC_EISCONN = 56,        // Already connected socket
+  SSC_ENOTCONN = 57,       // No connection for connection-type socket
+  SSC_ETIMEDOUT = 60,      // Timeout
+  SSC_ECONNREFUSED = 61,   // Connection request forcibly rejected
+  SSC_EHOSTUNREACH = 65,   // Remote host cannot be reached
+  SSC_EHOSTDOWN = 67,      // Remote host is down
+  SSC_EWOULDBLOCK = 70,    // Socket is in non-blocking mode and connection has not been completed
+  SSC_E_69 = 69,  // Socket is in non-blocking mode and a previously issued Connect command has not
+                  // been completed
+  SSC_SUCCESS = 70,
+};
+
+void Init(void);
+void FirmwareMap(bool on);
+u8* InitDIMM(u32 size);
+void InitKeys(u32 KeyA, u32 KeyB, u32 KeyC);
+u32 ExecuteCommand(std::array<u32, 3>& DICMDBUF, u32* DIIMMBUF, u32 Address, u32 Length);
+u32 GetGameType(void);
+u32 GetMediaType(void);
+bool GetTestMenu(void);
+void Shutdown(void);
```

Comment:
```suggestion
void Init();
void FirmwareMap(bool on);
u8* InitDIMM(u32 size);
void InitKeys(u32 KeyA, u32 KeyB, u32 KeyC);
u32 ExecuteCommand(std::array<u32, 3>& DICMDBUF, u32* DIIMMBUF, u32 Address, u32 Length);
u32 GetGameType();
u32 GetMediaType();
bool GetTestMenu();
void Shutdown();
```
Specifying `void` for an empty parameter list is unnecessary in C++. We largely don't do this in the rest of the codebase and for consistency I think it should be removed. Same goes at the definition of these functions.